edx_api/enrollments/__init__.py

```diff
@@ -36,6 +46,56 @@ def get_student_enrollments(self):
         resp.raise_for_status()
         return Enrollments(resp.json())
 
+    def get_enrollments(self, course_id=None, username=None, usernames=None):
+        """
+        List all course enrollments.
+
+        Args:
+            course_id (str, optional): If used enrollments will be filtered to the specified course id.
+            username (str, optional): Enrollments will be filtered to this specific username.
+            usernames (list, optional): List of usernames to filter enrollments.
+        """
+        params = {'page_size': 1}
+        if course_id is not None:
+            params['course_id'] = course_id
+        if username is not None:
+            params['username'] = username
+        if usernames is not None:
+            params['username'] = ','.join(params.get('username', []) + usernames)
```

Comment:
This might be simpler if we just only have a `usernames` parameter and assume it's always a list.
Callers can specify just one username in the list if they only want one.

Reply:
Sure, I'll update this.